libs/cart/src/effects/cart.effects.ts

```diff
@@ -52,4 +54,17 @@ export class DaffCartEffects {
         )
     )
   )
+
+  @Effect()
+  clearCart$ = this.actions$.pipe(
+    ofType(DaffCartActionTypes.CartResetAction),
+    switchMap(() =>
+      this.driver.clear().pipe(
+          map((resp) => new DaffCartResetSuccess(resp)),
```

Comment:
I believe this should be `return new DaffCartResetSuccess(resp)),`

Reply:
@Nolan-Arnold this is a short-hand equivalent of the same.

```
() => { return "string"; }
() => "string";
function(){
  return "string";
}
```

are all equivalent in `es6`